plugins/com.google.cloud.tools.eclipse.appengine.deploy/src/com/google/cloud/tools/eclipse/appengine/deploy/messages.properties

```diff
@@ -1,3 +1,7 @@
+dialog.staging.directory.provider.title=Select staging directory
+dialog.staging.directory.provider.message=Select a directory that will be used to export the exploded war file to run stage and deploy operations
+
 deploy.standard.dialog.title=Deploying to App Engine Standard
 deploy.standard.dialog.message=The project is being deployed to App Engine Standard (not really)
+deploy.standard.runnable.name=Deploying to App Engine Standard
```

Comment:
`deploy.standard.dialog.title` and `deploy.standard.dialog.message` removed as they are not used anymore.